staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -170,6 +170,9 @@ type KubeSchedulerProfile struct {
 // Enabled plugins are called in the order specified here, after default plugins. If they need to
 // be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
 type Plugins struct {
+	// Enqueue is a list of plugins that should be invoked before adding pods to the scheduling queue.
+	Enqueue PluginSet `json:"enqueue,omitempty"`
```

Comment:
is this a new capability? if so, does putting a value in this config field require a feature gate to be turned on?

Reply:
if so, docs need to indicate the feature gate connected to this

Reply:
We never used feature flags when adding new extensions since for the most part we consider them a refactor and a no-op. The schedulingGates feature (which is implemented using this new extension of the scheduler framework) itself is behind a feature gate.

Reply:
no, we don't disable the extension point itself no matter the featuregate is disabled or not. It's documented intentionally as a goal in the KEP for scheduler developers to use from day 1.